assets/js/index.js

```diff
@@ -3,7 +3,7 @@
 const generateLiTags = (gamesData, searchText = "") => {
   const liTags = [];
   searchText = searchText.trim().toLowerCase(); // Trim whitespace and convert to lowercase
-  let lengthOfJson = Object.keys(gamesData).length; // get the no. of games of Json to list them all
+  let lengthOfJson = Object.keys(gamesData).length+10; // get the no. of games of Json to list them all
```

Comment:
Remove 100 from the end if not needed

Reply:
@ishita-43 mam,  It is needed because due to some unknown issue games after 639 are not getting rendered.

Reply:
Ok, if this way it works fine we will approve

Reply:
done